backend/models/cans.py

```diff
@@ -740,3 +740,21 @@ class CAN(BaseModel):
     @BaseModel.display_name.getter
     def display_name(self):
         return self.number
+
+
+class CANFiscalYearFundingDetails(BaseModel):
+    """
+    The details of funding for a given fiscal year for a CAN.
+    """
+
+    __tablename__ = "can_fiscal_year_funding_details"
+
+    id: Mapped[int] = BaseModel.get_pk_column()
+    fund: Mapped[Optional[str]] = mapped_column(String(14))
```

Comment:
Please remove the String(*) declarations here - there's little gain from constraining the string's size and it could cause issues during the migration if the source data has issues

Reply:
right, FWIW in PG, varchar and text are implemented the same - unlike some DBs

Reply:
Just removed the length limit in the python model and the alembic scripts.